src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -357,19 +359,36 @@ func (cfg Config) doFetchCloudConfig(proxyAddr string) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("Unable to initialize HTTP client: %s", err)
 	}
+	country := strings.ToLower(globals.GetCountry())
```

Comment:
`GetCountry` returns the empty string in that case (`nil` is not a valid value for return value `string`).

https://github.com/getlantern/flashlight-build/blob/valencia/src/github.com/getlantern/flashlight/globals/globals.go#L34